docs/framework/20categories/30implementation/bug-bounty.md

```diff
@@ -0,0 +1,3 @@
+### Bug Bounty
```

Comment:
Is bug bounty an implementation or operational risk? This is also covered in part in the Operational Security Section. A separate section makes sense.